components/MapsDedi/index.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="wrapper">
+    <div class="maps-dedi">
+      <h4 class="maps-dedi__heading">
+        Data Perkembangan Desa Digital dan Mitra
+      </h4>
+      <p class="maps-dedi__subheading">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Enim aenean justo, adipiscing in lacinia mauris mauris vel.
+      </p>
+      <div class="maps-dedi__switch">
+        <div
+          :class="{
+            'maps-dedi__switch--active': isDesa,
+            'maps-dedi__switch--non-active': !isDesa,
+          }"
+          @click="joinSelected = 'Desa'"
+        >
+          <div
+            :class="{
+              'maps-dedi__switch--active-text': isDesa,
+              'maps-dedi__switch--non-active-text': !isDesa
+            }"
+          >
+            Desa Tergabung
+          </div>
+        </div>
+        <div
+          :class="{
+            'maps-dedi__switch--active': isMitra,
+            'maps-dedi__switch--non-active': !isMitra,
+          }"
```

Comment:
**open discussion:**
what's the difference if the code written like this
`isMitra ? 'maps-dedi__switch--active' : 'maps-dedi__switch--non-active'`

Reply:
there is the same goal but the different is the syntax
i use object binding class syntax and you show me ternary binding class syntax, so if i use ternary syntax would be
```javascript
:class="[isMitra ? 'maps-dedi__switch--active' : 'maps-dedi__switch--non-active']"
```

why I prefer object syntax because object syntax is easier to read and maintain for me when there are additional classes, e.g

```javascript
:class="[
    isMitra ? 'maps-dedi__switch--active' : 'maps-dedi__switch--non-active',
    {
        'maps-dedi__switch-normal'  : true,
    }
]"
```

cc: @yoslie

Reply:
i see, thanks for your perception 👍

Reply:
actually i agree the fact that not all ternary is readable tho